### PR TITLE
Fix vimtutor problem on Windows.

### PR DESCRIPTION
**Pull request by adah1972**
This change fixes issue https://github.com/vim/vim/issues/5756.

### CODE REVIEW

**Comment by k-takata** [comment]
It looks that this can close #2188.

File: vimtutor.bat [hunk 1]
```diff
@@ -10,7 +10,20 @@
 :: When that also fails, it uses the English version.
 
 :: Use Vim to copy the tutor, it knows the value of $VIMRUNTIME
-FOR %%d in (. "%TMP%" "%TEMP%") DO IF EXIST %%d\nul SET TUTORCOPY=%%d\$tutor$
+FOR %%d in (. %TMP% %TEMP%) DO (
+call :test_dir %0 %%d
+IF NOT ERRORLEVEL 1 GOTO dir_ok
```
Comment (k-takata) [inline vimtutor.bat]:
```suggestion
    call :test_dir %0 %%d
    IF NOT ERRORLEVEL 1 GOTO dir_ok
```
Indenting would be better.

File: vimtutor.bat [hunk 2]
```diff
@@ -10,7 +10,20 @@
...
+)
+
+echo No working directory is found
+GOTO end
+
+:test_dir
```
Comment (k-takata) [inline vimtutor.bat]: How about making the label more explanatory? (e.g. `:test_dir_writable`)
Or how about adding a comment?

File: vimtutor.bat (hunk 1 above)
Comment (adah1972) [reply vimtutor.bat]: I just followed the current convention. The other if-blocks are not indented, either. Should I indent just this block?

File: vimtutor.bat (hunk 2 above)
Comment (adah1972) [reply vimtutor.bat]: I think changing the label would be clearer. I prefer better names to comments.

File: vimtutor.bat (hunk 1 above)
Comment (k-takata) [reply vimtutor.bat]: Other if-commands are one-liners, not blocks, so they are not indented.
I think it's better to indent this for-block.

Comment (adah1972) [reply vimtutor.bat]: I see that I wrongly regarded some statements below an IF as a block—it is not, as the IF is a GOTO statement...

I'll change.

**Comment by adah1972** [comment]
@k-takata Let me know if there is anything missing.

**Comment by k-takata** [comment]
LGTM

**Pull request closed without merging**